LibreNMS/Exceptions/InsufficientDataException.php

```diff
@@ -14,17 +15,16 @@
  * GNU General Public License for more details.
  *
  * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  *
  * @package    LibreNMS
- * @link       https://www.librenms.org
- *
- * @author     Peca Nesovanovic <peca.nesovanovic@sattrakt.com>
+ * @link       http://librenms.org
+ * @copyright  2023 Tony Murray
```

Comment:
You've got 2023 in a few places just in case you want to update those to 2025.

Reply:
Well, you see... that's when those files were created :D

Reply:
Well yes but you've copied some files to new names and completely replaced the content.....:)

Reply:
Git is weird like that sometimes.  Create a new file and delete another file and it sometimes shows that as a "rename" even though the files were basically nothing alike.

Reply:
You learn something new every day, haven't seen that before.